packages/core/stories/button.doc.stories.mdx

```diff
@@ -111,32 +75,34 @@ export const Link = ({ children, to, ...props }) => (
 );
 
 <Canvas>
-  <ComponentAnatomy showLegend>
-    <Button aria-label="Simple Button">Default</Button>
-  </ComponentAnatomy>
-  <br />
-  <ComponentAnatomy>
-    <Button elementType="div" aria-label="Button as Div" variant="secondary">
-      Secondary Div
-    </Button>
-  </ComponentAnatomy>
-  <br />
-  <ComponentAnatomy>
-    <Button variant="cta" aria-label="CTA with Icon">
-      CTA with Icon <SendIcon size={12} />
-    </Button>
-  </ComponentAnatomy>
-  <br />
-  <ComponentAnatomy>
-    <Button
-      elementType={Link}
-      to="/"
-      aria-label="Link as elementType"
-      role={null}
-    >
-      Custom Link
-    </Button>
-  </ComponentAnatomy>
+  <Story name="HTML Anatomy">
+    <ComponentAnatomy showLegend>
```

Comment:
Not sure what anybody else thinks, but I've come to see the legend as superfluous - takes up space and adds noise without being very useful. I'd be inclined to drop it everywhere